articles/jakob-lind-learn-redux-by-coding-a-mini-redux/README.md

```diff
@@ -29,19 +29,19 @@
 Мы также должны определить редюсер и начальное состояние на момент запуска. Давайте с этого и начнём:
 
 ```js
-function createStore(reducer, initialState) {
-    var currentReducer = reducer;
-    var currentState = initialState;
+const createStore = (reducer, initialState) => {
+    const currentReducer = reducer;
+    let currentState = initialState;
 }
 ```
 
 ### 1. Получение состояния
 Итак, мы создали функцию, которая просто сохраняет начальное состояние и редюсер как локальные переменные. Сейчас давайте реализуем возможность получать состояние нашего хранилища.
 
 ```js
-function createStore(reducer, initialState) {
-    var currentReducer = reducer;
-    var currentState = initialState;
+const createStore (reducer, initialState) => {
```

Comment:
```suggestion
const createStore = (reducer, initialState) => {
```

Reply:
What a shame. 🤦‍♂️ This is my inattention.